.github/workflows/ml-kem.yml

```diff
@@ -23,32 +23,33 @@ jobs:
     with:
       msrv: 1.74.0
 
-# TODO
-#  no_std:
-#    needs: set-msrv
-#    runs-on: ubuntu-latest
-#    strategy:
-#      matrix:
-#        rust:
-#          - ${{needs.set-msrv.outputs.msrv}}
-#          - stable
-#        target:
-#          - thumbv7em-none-eabi
-#          - wasm32-unknown-unknown
-#    steps:
-#      - uses: actions/checkout@v4
-#      - uses: RustCrypto/actions/cargo-cache@master
-#      - uses: dtolnay/rust-toolchain@master
-#        with:
-#          toolchain: ${{ matrix.rust }}
-#          targets: ${{ matrix.target }}
-#      - run: cargo build --no-default-features --target ${{ matrix.target }}
+  no_std:
+    needs: set-msrv
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust:
+          - ${{needs.set-msrv.outputs.msrv}}
+          - stable
+        target:
+          # XXX(bifurcation) We have a dependency on `sha3`, which has a
+          # transitive `std` dependency by default.  It appears that we can turn
+          # this off via Cargo.
```

Comment:
This should fix it: https://github.com/RustCrypto/KEMs/pull/5#discussion_r1509997687

Reply:
Ah, before I saw this, I tried copy/pasting [the approach `sha3` seems to take](https://github.com/RustCrypto/hashes/blob/master/sha3/Cargo.toml#L28), passing through an `std` feature but not otherwise using it.  If that doesn't work, I'll fall back to this suggestion.

Reply:
Since `std` is a default feature of `sha3`, you need to disable it.

I merged the suggestion before seeing this comment, but it appears to have fixed the issue.

Reply:
Ah, good. This solution seems simpler anyway.